patroni/api.py

```diff
@@ -180,6 +180,80 @@ def do_GET_config(self):
         else:
             self.send_error(502)
 
+    def do_GET_metrics(self):
+        postgres = self.get_postgresql_status(True)
+        patroni = self.server.patroni
+
+        metrics = []
+
+        metrics.append("# HELP patroni_version Patroni semver without periods.")
+        metrics.append("# TYPE patroni_version gauge")
+        padded_semver = ''.join([x.zfill(2) for x in patroni.version.split('.')])  # 2.0.2 => 020002
+        metrics.append("patroni_version {0}".format(padded_semver).replace('.', ''))
```

Comment:
Looks like `replace()` is redundant.
```suggestion
        metrics.append("patroni_version {0}".format(padded_semver))
```